actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -178,6 +182,16 @@ def _filtered_referrer # :nodoc:
       #   link_to("Destroy", "http://www.example.com", method: :delete)
       #   # => <a href='http://www.example.com' rel="nofollow" data-method="delete">Destroy</a>
       #
+      # If you use <tt>target: "_blank"</tt>, the <tt>rel="noopener"</tt> will added as default.
+      #
+      #   link = link_to("Hello", "http://www.example.com", target: "_blank")
+      #   # => <a target="_blank" rel="noopener" href="http://www.example.com">Hello</a>
+      #
+      # but you can disable <tt>rel="noopener"</tt> with using <tt>opener: true</tt>.
+      #
+      #   link = link_to("Hello", "http://www.example.com", target: "_blank")
+      #   # => <a target="_blank" rel="noopener" href="http://www.example.com">Hello</a>
```

Comment:
Should this example have opener true like: `link = link_to("Hello", "http://www.example.com", target: "_blank", opener: true)`? Right now this example is the same as the first example.